mllib/src/main/scala/org/apache/spark/ml/feature/StopWordsRemover.scala

```diff
@@ -84,7 +86,28 @@ class StopWordsRemover @Since("1.5.0") (@Since("1.5.0") override val uid: String
   @Since("1.5.0")
   def getCaseSensitive: Boolean = $(caseSensitive)
 
-  setDefault(stopWords -> StopWordsRemover.loadDefaultStopWords("english"), caseSensitive -> false)
+  /**
+   * Locale of the input for case insensitive matching. Ignored when [[caseSensitive]]
+   * is true.
+   * Default: Locale.getDefault.toString
```

Comment:
I understand that Locale.getDefault is the current behavior, yet since we're using english as default stopwords, I'm leaning towards using English as default locale.
Also using Locale.getDefault means that the same code will behave differently on different nodes, which adds extra complexity for trouble-shooting.

Reply:
Locale is usually configured at system level. Setting the default locale to `en` is a breaking change to users who set system locale to `tr` and have the following code:

~~~
val remover = new StopWordsRemover()
  .setStopWords(loadDefaultStopWords("turkish")
~~~